packages/fx-core/.gitignore

```diff
@@ -9,3 +9,6 @@ coverage
 templates/plugins/resource/bot/**/*.zip
 templates/plugins/resource/frontend/**/*.zip
 templates/plugins/resource/function/**/*.zip
+
+# test
+tests/plugins/solution/testProject/infra/azure/templates/main.json
```

Comment:
why we ignore this file?

Reply:
Will remove this file. This file is the output file of bicep cli.